app/client/components/cancel/cancellationFlow.tsx

```diff
@@ -58,15 +59,15 @@ class ReasonPicker extends React.Component<
           <select
             value={this.state.reasonPath}
             onChange={this.handleChange}
-            css={{
+            css={css({
               ...cssInheritFont,
               width: "100%",
               height: "32px",
               border: "1px black solid",
-              color: this.state.reasonPath ? undefined : palette.neutral["4"],
-              appearance: "menulist"
+              color: this.state.reasonPath ? undefined : palette.neutral["4"]
+              // appearance: "menulist"
```

Comment:
Are these commented out lines still required?

Reply:
this one is because the typecheck fails, and not sure its adding anything, but don't want to loose it just yet as I will be revisiting some of the stuff in this PR in the near future. In fact, kinda need to re-write this component, currently uses a `<select>` which is terrible on iOS (so want to rebuild as just a repeatable clickable `div`).